observations/io/io_knmi.py

```diff
@@ -155,18 +196,59 @@ def _start_end_to_datetime(start, end):
     """
 
     if start is None:
-        start = pd.Timestamp(pd.Timestamp.today().year, 1, 1)
+        start = pd.Timestamp(pd.Timestamp.today().year-1, 1, 1)
     else:
         start = pd.to_datetime(start)
     # start date one day before because later the datetime index is modified
     start = start - pd.Timedelta(1, 'D')
     if end is None:
-        end = pd.Timestamp.today()
+        end = pd.Timestamp.today() - pd.Timedelta(1, unit='D')
     else:
         end = pd.to_datetime(end)
 
     return start, end
 
+def _check_latest_measurement_date_RD_debilt(verbose=False):
+    """ According to the website of the knmi it can take up to 3 weeks before
+    precipitation data is updated. If you use the fill_missing_measurements
+    method to fill a time series untill today, it will keep looking at all
+    stations to find the data of these last days that probably does not exist.
+    
+    To prevent this, this function will find the last day there are measure-
+    ments at knmi station de Bilt. It is assumed that if de Bilt doesn't have
+    recent measurements, no station will have measurements for these dates.
+    
+    website knmi: https://www.knmi.nl/nederland-nu/klimatologie/monv/reeksen
+    
+    Parameters
+    ----------
+    verbose : boolean, optional
+        Print additional information to the screen (default is False).
+
+    Returns
+    -------
+    last_measurement_date_debilt : pd.TimeStamp
+        last date with measurements at station de Bilt
+
+    """
+    
+    url = 'http://projects.knmi.nl/klimatologie/monv/reeksen/getdata_rr.cgi'
+    start = pd.datetime.now() - pd.Timedelta(21, unit='D')
+    knmi_df, variables = get_knmi_daily_rainfall(url, 550, "RD", None, None, 
+                                                 False, verbose=verbose)
+    knmi_df = knmi_df.dropna()
+    if knmi_df.empty:
+        raise ValueError('knmi station de Bilt has no RD measurements in the past 3 weeks. HELP!')
```

Comment:
Maybe removed the HELP :P!

Reply:
Yes I will, maybe it is not that scary :)